src/main.rs

```diff
@@ -0,0 +1,116 @@
+//! Actix Basic
```

Comment:
Let's work on code organization.  Only have minimal files in this folder.  Place your handlers (controller functions) in a separate folder `/handlers` and your current user.rs file in `/models/user.rs`.

Reply:
hm... ran into issues with this in the most recent commit. Can you peek at why tests run *twice*? (I will get them working)